exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/AnyValueMarshaler.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.exporter.otlp.internal;
+
+import io.opentelemetry.proto.common.v1.internal.AnyValue;
+import java.io.IOException;
+
+public class AnyValueMarshaler extends MarshalerWithSize {
```

Comment:
Apparently it shouldn't have been a private class - we have this marshaler already here, can move up

https://github.com/open-telemetry/opentelemetry-java/blob/main/exporters/otlp/common/src/main/java/io/opentelemetry/exporter/otlp/internal/KeyValueMarshaler.java#L111

(because it's a oneof, each anyvalue gets its own marshaler)